### PR TITLE
Prepare first release

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

**Comment by sven1103** [comment]
[Refactor tracking helper library](https://trello.com/c/oQ3OojQm/42-refactor-tracking-helper-library)

File: test/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 QBiC
```
Comment (Zethson) [inline test/LICENSE]: 2020? :)

Comment (wow-such-code) [reply test/LICENSE]: wrong year!

**Review by wow-such-code — CHANGES_REQUESTED** [review]
change year in test/.gitignore

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**